templates/address.html

```diff
@@ -0,0 +1,7 @@
+<h1>Address {{ self.address }}</h1>
+<h2>Outputs</h2>
+<ul>
+%% for output in self.outputs.iter() {
+  <li><a class=monospace href=/output/{{ output }}>{{ output }}</a></li>
+%% }
+</ul>
```

Comment:
This is useful for a backend like the API response. But as a front end user can we maybe paginate something that previews what actually is on the output? Like runes, inscriptions and BTC balance.

Reply:
We're just doing outputs initially to keep the PR simple. Follow-up PRs can add btc/inscriptions/runes.